docs/application/dotnet/guides/user-interface/nui/ui-components.md

```diff
@@ -11,29 +11,33 @@ The following table lists the available UI components:
+| [DatePicker](./nui-components/DatePicker.md)        | <div align="left">DatePicker is a class which provides a function that allows the user to select a date through a scrolling motion by expressing the specified value as a list. DatePicker expresses the current date using the locale information of the system. Year range is 1970~2038 (glibc time_t struct min, max value) <br>[Guide ](./nui-components/DatePicker.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.DatePicker.html)  | <img src="./screenshots/DatePicker.png" width="300"> <br>[C# code for this page](./codeExamples/DatePicker.cs) / [XAML page](./codeExamples/DatePicker.xaml) |
+| [ImageView](./imageview.md)          | An image view is a class for displaying an image resource.  <br>[Guide ](./nui-components/ImageView.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.BaseComponents.ImageView.html)  | <img src="./screenshots/ImageView.png" width="300"> <br>[C# code for this page](./codeExamples/ImageView.cs) / [XAML page](./codeExamples/ImageView.xaml) |
+| [Loading](./nui-components/Loading.md) | A loading is used to give information about the ongoing operations. <br>[Guide ](./nui-components/Loading.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.Loading.html) | <img src="./nui-components/media/loading.png" width="300"> <br>[C# code for this page](./codeExamples/Loading.cs) / [XAML page](./codeExamples/Loading.xaml) |
+| [LottieAnimationView](./nui-components/LottieAnimationView.md) | A LottieAnimationView is used to show a vector animation. <br>[Guide ](./nui-components/LottieAnimationView.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.BaseComponents.LottieAnimationView.html) | <img src="./screenshots/LottieAnimationView.png" width="300"> <br>[C# code for this page](./codeExamples/LottieAnimationView.cs) / [XAML page](./codeExamples/LottieAnimationView.xaml) |
+| [Menu](./nui-components/Menu.md)        | <div align="left">Menu is a class which contains a set of MenuItems and has one of them selected.</div> <br>[Guide ](./nui-components/Menu.md) / [API Documentation](https://docs.tizen.org/application/dotnet/api/TizenFX/API9/api/Tizen.NUI.Components.Menu.html) | <img src="./screenshots/Menu.png" width="300"> <br>[C# code for this page](./codeExamples/Menu.cs) / [XAML page](./codeExamples/MenuTestPage.xaml) <br> [C# code Menu definition](./codeExamples/MenuTestMenu.xaml.cs) / [XAML Menu definition](./codeExamples/MenuTestMenu.xaml) |
```

Comment:
Line 23: Menu 
The use of **< div >** tag is causing a line break between the description and the “Guide and API Documentation” hyperlinks. This is creating an inconsistency with the rest of the table format.